.github/workflows/tests.yml

```diff
@@ -6,7 +6,13 @@ on:
 
 jobs:
   tests:
-    runs-on: ubuntu-latest
+    strategy:
```

Comment:
I've never done this before and got a bit curious how this works so I thought I'd give it a go. Let's see how it works out.